modules/back-up-quay-procedure.adoc

```diff
@@ -0,0 +1,191 @@
+== Backing up Red Hat Quay
+
+This procedure is exclusively for OpenShift Container Platform and NooBaa deployments.
+
+.Prerequisites
+
+* A {productname} deployment on OpenShift Container Platform.
+* An initial user set as `superuser` to prevent user creation.
+* At least two user accounts. This procedure uses `test1` and `test2` as sample user names.
+* A organization within your Quay deployment. This procedure uses `testorg` as its example organization.
+* An image in your repository. This documentation uses the `busybox` image for backing up and restoring.
+
+.Procedure
+
+[NOTE]
+====
+The procedure in this documents uses the following namespace: `quay-enterprise`.
+====
+
+. Obtain a list of your deployed pods in your Quay namespace:
++
+----
+$ oc get pods -n quay-enterprise
+----
++
+Example output:
++
+----
+NAME                                                   READY   STATUS      RESTARTS   AGE
+example-registry-clair-app-7dccddc6fd-fwckp            1/1     Running     0          69m
+example-registry-clair-postgres-558d7cc9c-xgk2m        1/1     Running     0          68m
+example-registry-quay-app-5677cb59b7-fpxcg             1/1     Running     0          69m
+example-registry-quay-app-upgrade-zstr4                0/1     Completed   0          69m
+example-registry-quay-config-editor-67c769f669-dp5gf   1/1     Running     0          69m
+example-registry-quay-database-6d58c4c545-99s2g        1/1     Running     0          69m
+example-registry-quay-mirror-d4bc9c99c-rmk4f           1/1     Running     0          68m
+example-registry-quay-postgres-init-rwbsq              0/1     Completed   0          69m
+example-registry-quay-redis-577776d494-k48zv           1/1     Running     0          69m
+quay-operator.{productminv}-798d5c8b69-q6x88                  1/1     Running     0          79m
+----
+
+. Obtain the configuration secret of your Quay application deployment in JSON:
++
+----
+$ oc get deployment -n quay-enterprise example-registry-quay-app -o json | jq '.spec.template.spec.containers[].env[]'
+----
++
+Example output:
++
+[source,terminal]
+----
+{
+  "name": "QE_K8S_CONFIG_SECRET",
+  "value": "example-registry-quay-config-secret-m25mf4dg78"
+}
+{
+  "name": "QE_K8S_NAMESPACE",
+  "valueFrom": {
+    "fieldRef": {
+      "apiVersion": "v1",
+      "fieldPath": "metadata.namespace"
+    }
+  }
+}
+{
+  "name": "DEBUGLOG",
+  "value": "false"
+}
+{
+  "name": "WORKER_COUNT_WEB",
+  "value": "4"
+}
+{
+  "name": "WORKER_COUNT_SECSCAN",
+  "value": "2"
+}
+{
+  "name": "WORKER_COUNT_REGISTRY",
+  "value": "8"
+}
+----
+
+. Back up the Quay configuration secret into a yaml file, for example `quay-config-yaml.backup.yaml`:
++
+----
+$ oc get secret -n quay-enterprise example-registry-quay-config-secret-m25mf4dg78 -o json | jq '.data."config.yaml"' | cut -d '"' -f2 | base64 -d -w0 > quay-config-yaml-backup.yaml
+----
+
```

Comment:
Do we first need to quiesce (allows existing connections to complete without being severed and disallows any new connections to a server or service) the traffic; and stop the Quay Services first, so that the backup that we take in the next step is static. 

- Note: its a this point the Quay service would take downtime.

Reply:
`$ oc scale deployment example-registry-restore-quay-app --replicas=0 -n quay-enterprise-restore`

We document this in the restore part of the docs.